Parity/prminput/prex_maindet_eventcuts.3453-3582.map

```diff
@@ -1,7 +1,8 @@
 ! For runs 3564 and 3565 (HV off left, Quads off Right) - want to local cut the main RHRS detectors, global cut the main LHRS detectors
+! left arm running of slug 8-11
 
 EVENTCUTS = 3
-IntegrationPMT, usl,		-1000.000,	100000,	g,	0
-IntegrationPMT, dsl,		-1000.000,	100000,	g,	0
+IntegrationPMT, usl,		0.015,	100000,	g,	0
```

Comment:
I recall at one point during the experiment we accidentally put a low cut similar to this that was actually too high (and voided the data for an hour while we tried to figure it out). 

Anyway, if this low level is in fact safe then this is fine, but I just wanted to make sure that you're aware of the time that I made a big mistake on this kind of cut :)